test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -35,13 +35,52 @@ internal static class RegressionTest
 
         private static int Main(string[] args)
         {
-            return Parser.Default.ParseArguments<Options>(args).MapResult(
-                Run,
-                _ =>
+            if (args.Length > 1 && args[0].Equals("warm-up"))
+            {
+                Console.WriteLine($"warm up starting...");
+
+                var opt = string.Join(" ", args[1..]);
+
+                if (string.IsNullOrEmpty(opt))
                 {
-                    SendPullRequestComments(new() { CrashMessage = "regression-test argument exception" });
-                    return -9999;
-                });
+                    throw new InvalidDataException();
+                }
+
+                try
+                {
+                    var option = Parser.Default.ParseArguments<Options>($"{opt}".Split()).MapResult(WarmUpAgents, _ => { return -9999; });
+                }
+                catch
+                {
+                    Console.WriteLine($"Clone failed: {opt}");
+                }
+
+                return 0;
+            }
+            else
+            {
+                Console.WriteLine("run regression test starting...");
+                return Parser.Default.ParseArguments<Options>(args).MapResult(
+                    Run,
+                    _ =>
+                    {
+                        SendPullRequestComments(new() { CrashMessage = "regression-test argument exception" });
+                        return -9999;
+                    });
+            }
+        }
+
+        private static int WarmUpAgents(Options opts)
```

Comment:
Remove this method (this has many duplicated code with the `Run` method), instead, return after `EnsureTestData` if `opt.WarmUp` is true.

Reply:
Fixed.